app/views/api_docs/reference/v0.1/_reference.md

```diff
@@ -1237,14 +1237,14 @@ Withdraw a trainee.
 
 There is no request body for this endpoint.
 
-Note that multiple values for the reasons parameter can be provided by repeating the parameter in the query string, e.g. `reasons[]=personal_reasons&reasons[]=got_a_job`
+Note that multiple values for the reasons parameter can be provided by repeating the parameter in the query string, e.g. `reasons[]=family_problems&reasons[]=got_a_job`
 
 #### Parameters
 
 | **Parameter** | **In**  | **Type** | **Required** | **Description** |
 | ------------- | ------- | -------- | ------------ | --------------- |
 | **trainee_id** | path | string | true | The unique ID of the trainee |
-| **reasons** | query | array of strings | true | The reason(s) for the withdrawal |
+| **reasons** | query | array of strings | true | The reason(s) for the withdrawal. Valid values are `could_not_give_enough_time`, <br>`course_was_not_suitable`,<br> `did_not_make_progress`,<br> `did_not_meet_entry_requirements`,<br> `does_not_want_to_become_a_teacher`,<br> `family_problems`,<br> `financial_problems`,<br> `got_a_job`,<br> `problems_with_their_health`,<br> `stopped_responding_to_messages`,<br> `teaching_placement_problems`,<br> `unacceptable_behaviour`,<br> `unhappy_with_course_provider_or_employing_school`,<br> `another_reason`,<br> `unknown` |
```

Comment:
Yeah, it is [not particularly clear](https://github.com/DFE-Digital/register-trainee-teachers/blob/7187-validate-withdrawal-reasons-against-current-list-of-reasons-and-add-to-docs/app/views/trainees/withdrawal/reasons/edit.html.erb#L44) as you say, but I think the correct list is `WithdrawalReasons::REASONS` which matches the list in the web interface.
As with other parts of these docs, there will be a certain amount of manual updating required. It would be lovely to generate lists like this from the code but I think that would require a significant shift away from markdown.